### ISSUE TITLE
BUG: compatibility with `plotly.express.choropleth`

### ISSUE THREAD

**Issue by Borda** [modin-project/modin#6770]
### Modin version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the latest released version of Modin.

- [ ] I have confirmed this bug exists on the main branch of Modin. (In order to do this you can follow [this guide](https://modin.readthedocs.io/en/stable/getting_started/installation.html#installing-from-the-github-master-branch).)

### Reproducible Example

```python
df = pd.read_csv("/kaggle/input/united-nations-refugee-data-unhcr/United Nations Refugee Data.csv")

df_ = df.groupby(["Year", "Country of origin (ISO)"])[["Refugees under UNHCR's mandate"]].sum().reset_index()

import plotly.express as px

fig_world_map = px.choropleth(
    data_frame=df_,  # ._to_pandas()
    locations="Country of origin (ISO)",
    color="Refugees under UNHCR's mandate",
    animation_frame="Year",
)
```

### Issue Description

the plot just crash

### Expected Behavior

showing interactive map
when passing `data_frame=df_._to_pandas()` helps

### Error Logs

<details>

```python-traceback
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[16], line 1
----> 1 fig_world_map = px.choropleth(
      2     data_frame=df_,  # ._to_pandas()
      3     locations="Country of origin (ISO)",
      4     color="Refugees under UNHCR's mandate",
      5     animation_frame="Year",
      6     title="Refugees Origine Over Time",
      7     range_color=[0, int(max_all_years)],
      8     color_continuous_scale="Reds",
      9     labels={'Refugees under UNHCR\'s mandate': '#Refugees',
     10             'Country of origin (ISO)': 'Origin'},
     11     projection="natural earth",
     12     category_orders={"Year": sorted(df_["Year"].unique())}
     13 )
     15 fig_world_map.update_layout(height=800, width=1200)
     16 fig_world_map.show()

File /opt/conda/lib/python3.10/site-packages/plotly/express/_chart_types.py:1091, in choropleth(data_frame, lat, lon, locations, locationmode, geojson, featureidkey, color, facet_row, facet_col, facet_col_wrap, facet_row_spacing, facet_col_spacing, hover_name, hover_data, custom_data, animation_frame, animation_group, category_orders, labels, color_discrete_sequence, color_discrete_map, color_continuous_scale, range_color, color_continuous_midpoint, projection, scope, center, fitbounds, basemap_visible, title, template, width, height)
   1051 def choropleth(
   1052     data_frame=None,
   1053     lat=None,
   (...)
   1085     height=None,
   1086 ) -> go.Figure:
   1087     """
   1088     In a choropleth map, each row of `data_frame` is represented by a
   1089     colored region mark on a map.
   1090     """
-> 1091     return make_figure(
   1092         args=locals(),
   1093         constructor=go.Choropleth,
   1094         trace_patch=dict(locationmode=locationmode),
   1095     )

File /opt/conda/lib/python3.10/site-packages/plotly/express/_core.py:2076, in make_figure(args, constructor, trace_patch, layout_patch)
   2073 layout_patch = layout_patch or {}
   2074 apply_default_cascade(args)
-> 2076 args = build_dataframe(args, constructor)
   2077 if constructor in [go.Treemap, go.Sunburst, go.Icicle] and args["path"] is not None:
   2078     args = process_dataframe_hierarchy(args)

File /opt/conda/lib/python3.10/site-packages/plotly/express/_core.py:1485, in build_dataframe(args, constructor)
   1482     args["color"] = None
   1483 # now that things have been prepped, we do the systematic rewriting of `args`
-> 1485 df_output, wide_id_vars = process_args_into_dataframe(
   1486     args, wide_mode, var_name, value_name
   1487 )
   1489 # now that `df_output` exists and `args` contains only references, we complete
   1490 # the special-case and wide-mode handling by further rewriting args and/or mutating
   1491 # df_output
   1493 count_name = _escape_col_name(df_output, "count", [var_name, value_name])

File /opt/conda/lib/python3.10/site-packages/plotly/express/_core.py:1210, in process_args_into_dataframe(args, wide_mode, var_name, value_name)
   1208         if argument == "index":
   1209             err_msg += "\n To use the index, pass it in directly as `df.index`."
-> 1210         raise ValueError(err_msg)
   1211 elif length and len(df_input[argument]) != length:
   1212     raise ValueError(
   1213         "All arguments should have the same length. "
   1214         "The length of column argument `df[%s]` is %d, whereas the "
   (...)
   1221         )
   1222     )

ValueError: Value of 'animation_frame' is not the name of a column in 'data_frame'. Expected one of ["Refugees under UNHCR's mandate", 'Country of origin (ISO)'] but received: Year
```

</details>

### Installed Versions

<details>

```
INSTALLED VERSIONS
------------------
commit              : b99cf06a0d3bf27cb5d2d03d52a285cc91cfb6f6
python              : 3.10.12.final.0
python-bits         : 64
OS                  : Linux
OS-release          : 5.15.133+
Version             : #1 SMP Wed Nov 8 17:30:28 UTC 2023
machine             : x86_64
processor           : x86_64
byteorder           : little
LC_ALL              : C.UTF-8
LANG                : C.UTF-8
LOCALE              : en_US.UTF-8

Modin dependencies
------------------
modin               : 0.25.1
ray                 : 2.6.3
dask                : 2023.11.0
distributed         : 2023.11.0
hdk                 : None

pandas dependencies
-------------------
pandas              : 2.1.3
numpy               : 1.24.3
pytz                : 2023.3
dateutil            : 2.8.2
setuptools          : 68.1.2
pip                 : 23.2.1
Cython              : 3.0.0
pytest              : 7.4.3
hypothesis          : None
sphinx              : None
blosc               : None
feather             : 0.4.1
xlsxwriter          : None
lxml.etree          : 4.9.3
html5lib            : 1.1
pymysql             : None
psycopg2            : None
jinja2              : 3.1.2
IPython             : 8.14.0
pandas_datareader   : 0.10.0
bs4                 : 4.12.2
bottleneck          : None
dataframe-api-compat: None
fastparquet         : None
fsspec              : 2023.10.0
gcsfs               : 2023.6.0
matplotlib          : 3.7.3
numba               : 0.57.1
numexpr             : 2.8.7
odfpy               : None
openpyxl            : 3.1.2
pandas_gbq          : None
pyarrow             : 14.0.1
pyreadstat          : None
pyxlsb              : None
s3fs                : 2023.10.0
scipy               : 1.11.3
sqlalchemy          : 2.0.20
tables              : 3.9.1
tabulate            : 0.9.0
xarray              : 2023.10.1
xlrd                : None
zstandard           : 0.22.0
tzdata              : 2023.3
qtpy                : None
pyqt5               : None
```

</details>

**Comment by anmyachev** [comment]
Hello @Borda, thanks for the contribution! Which version of `plotly` are you using?

**Comment by Borda** [comment]
> Hello, thanks for the contribution! Which version of `plotly` are you using?

```
plotly                                   5.16.1
plotly-express                           0.4.1
```

**Comment by anmyachev** [comment]
I can reproduce the problem. I'll see how to fix this.

**Comment by anmyachev** [comment]
https://github.com/plotly/plotly.py/pull/4442

**Comment by anmyachev** [comment]
@Borda the pull request in `plotly.py` has been merged. It will be available, I believe, from the `v5.19.0` version.

**Comment by Borda** [comment]
> @Borda the pull request in `plotly.py` has been merged. It will be available, I believe, from the `v5.19.0` version.

Thank you!
